src/evolutionary_keras/models.py

```diff
@@ -98,23 +101,55 @@ def perform_genetic_fit(
                 information = f" > epoch: {epoch+1}/{epochs}, {loss} "
                 log.info(information)
 
+            epoch_logs.update(score)
```

Comment:
Why the use of update here? As far as I can see you get the same results using `score` instead of `epoch_logs` all the way through. Maybe I'm missing something.

Reply:
To be honest, I just looked at what is done in tf: https://github.com/tensorflow/tensorflow/blob/v2.3.0/tensorflow/python/keras/engine/training.py#L824-L1146

But I expect that the the reason loss-history is also stored in the callback logs, is in case it is needed for something (I'm thinking of e.g. stopping). I'm not sure what kind of use the the callback at the end of the training can have, but also there I'd imagine the history could be of importance rather than just the final epoch (e.g. in case callback is interested in the best result).

Reply:
I see, from what I see in that file the loop is over the possible output layers of the model, so you want to update so that the dictionary stores a reference to each output.
In this case you are looping over the number of epochs so the score dictionary will always have the same entries (so update is just replacing the full dictionary each time)